dockerfiles/Caffe2Dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG CODE_VERSION
+FROM clipper/py-rpc:${CODE_VERSION}
+
+COPY clipper_admin/clipper_admin/python_container_conda_deps.txt /lib/
+
+RUN echo deb http://ftp.de.debian.org/debian jessie-backports main >> /etc/apt/sources.list \
+      && apt-get update --fix-missing \
+      && apt-get install -yqq -t jessie-backports openjdk-8-jdk \
+      && conda install -y --file /lib/python_container_conda_deps.txt \
+      && conda install -c anaconda cloudpickle=0.5.2 \
+      && conda install -c ezyang onnx\
+      && conda install -c conda-forge protobuf==3.4.0 \
+      && conda install -c ezyang/label/devgpu caffe2 \
```

Comment:
This should be `&& conda install -c caffe2 caffe2 \`

Reply:
I tried this command, but it seems to conflict with other packages in my local environment, so I install another one from anaconda cloud. I will change them as you say to see whether test can pass.  @dcrankshaw